<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>901</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QGraphicsView" name="inputImg">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>271</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="ourImg">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>50</y>
        <width>271</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="calcTimeO">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>390</y>
        <width>271</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Temps calcul</string>
      </property>
     </widget>
     <widget class="QPushButton" name="fileButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Choisir fichier</string>
      </property>
     </widget>
     <widget class="QLabel" name="simO">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>420</y>
        <width>271</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Ressemblance</string>
      </property>
     </widget>
     <widget class="QLabel" name="Oracle">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>10</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Oracle</string>
      </property>
     </widget>
     <widget class="QLabel" name="Us">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>20</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Nous</string>
      </property>
     </widget>
     <widget class="QLabel" name="fileLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom fichier</string>
      </property>
     </widget>
     <widget class="QLabel" name="calcTimeUs">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>390</y>
        <width>271</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Temps calcul</string>
      </property>
     </widget>
     <widget class="QLabel" name="simUs">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>420</y>
        <width>271</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Ressemblance</string>
      </property>
     </widget>
     <widget class="QPushButton" name="startButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>380</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Chercher ressemblances</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="oracleImg">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>50</y>
        <width>271</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
